migrations/1641482376890-add-overrides-to-subscriber-info.js

```diff
@@ -0,0 +1,22 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  var sql = [
+    `ALTER TABLE customers 
+     ADD COLUMN phone_override VERIFICATION_TYPE NOT NULL DEFAULT 'automatic',
+     ADD COLUMN phone_override_by UUID,
+     ADD COLUMN phone_override_at TIMESTAMPTZ
+    `,
+    `ALTER TABLE edited_customer_data
+     ADD COLUMN phone TEXT,
+     ADD COLUMN phone_at TIMESTAMPTZ,
+     ADD COLUMN phone_by UUID
+    `
```

Comment:
Isn't phone number the one field we can't edit?

Reply:
yes, but it's disabled on the admin level. I left this logic here since we'll use it in the future, right?

Reply:
I don't think we will. Also, if we do that for the phone it might have another structure in place, since that's kinda the ID for the customers right now.